real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/nodes/RBSMNode.java

```diff
@@ -163,18 +245,26 @@ public static JSONObject translatePeelMessageToJObject(String message) {
 		
 		data.put("timestamp", messageData[1]);
 		
-		if(sensorName.equals("steering") || sensorName.equals("commanded_steering")) {
+		if(sensorName.equals("steering") || sensorName.equals("commanded_steering")) 
+		{
 			data.put("name", "Steering_" + sensorName);
 			params.put("angle", Float.valueOf(messageData[2]));
 		}
-		else if (sensorName.equals("encoder")) {
+		else if (sensorName.equals("fp_hash")) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Position literals first in String comparisons](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/1384444228/issues/source?bid=2938665&fileBranchId=2938741#l253)